MapboxNavigation/NavigationMapView.swift

```diff
@@ -437,7 +437,7 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         let line = MGLPolyline(coordinates: coords, count: UInt(coords.count))
         let camera = cameraThatFitsShape(line, direction: direction, edgePadding: padding)
         
-        setCamera(camera, animated: false)
+        setCamera(camera, animated: animated)
```

Comment:
nice catch, I wonder if there was a reason this was explicitly set to `false` before?

Reply:
Probably an oversight. But all the code paths currently call this method with `false` for `animated`, so the change has no actual effect.